tests/aeroval/test_aeroval_HIGHLEV.py

```diff
@@ -102,21 +103,6 @@ def test_reanalyse_existing(eval_config: dict):
     assert output.is_dir()
     assert list(output.glob("**/*.nc"))
 
-    cfg.colocation_opts.reanalyse_existing = False
-    proc = ExperimentProcessor(cfg)
-    assert proc.reanalyse_existing == False
-
-    proc.run()
-    assert output.is_dir()
-    assert list(output.glob("**/*.nc"))
-
-    proc.exp_output.delete_experiment_data(also_coldata=True)
-    assert not output.exists()
-
-    proc.run()
-    assert output.is_dir()
-    assert list(output.glob("**/*.nc"))
-
```

Comment:
are this `asserts` no longer relevant?

Reply:
No, these tests were overly verbose and the functionality covered by reparamterizing the test to accept a boolean `reanalyse_existing`. 

This tests highlights one of the "bad practices" which only really happens in the tests as far as I can tell, but with the current setup of this Pydantic-based implementation, once the config is defined, it is immutable. So one can not change attributes in `cfg` like this anymore by setting them. They will not change. You must change the actual config dictionary prior to giving it to `EvalSetup`, and hence these tests were broken and needed to be rewritten.